modules/ztp-troubleshooting-gitops-ztp.adoc

```diff
@@ -0,0 +1,8 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-zero-touch-provisioning.adoc
+
+[id="ztp-troubleshooting-gitops-ztp_{context}"]
+= Troubleshooting GitOps ZTP
+
+As noted, the ArgoCD pipeline synchronizes the SiteConfig and PolicyGenTemplate custom resources (CRs) from the Git repository to the hub cluster. During this process, post-sync hooks create the installation and policy CRs that are also applied to the hub cluster. Use the following procedures to troubleshoot issues that might occur in this process.
```

Comment:
s/SiteConfig and PolicyGenTemplate/`SiteConfig` and `PolicyGenTemplate`

"The ArgoCD pipeline synchronizes the `SiteConfig` and `PolicyGenTemplate` custom resources (CRs) from the Git repository to the hub cluster."

Reply:
changed